### ISSUE TITLE
"Friday" is shown twice instead of "Thursday" and "Friday"

### ISSUE THREAD

**Issue by erkexzcx** [rand256/valetudo#55]
TL;DR: https://github.com/rand256/valetudo/blob/aef814870272bc2cb5e557519130657cab77abcc/client/settings-timers.html#L66 (CTRL+F and search for `class="center" data-i18n="common.days.4">`, then cycle through results)

________

In provided latest OS image, there is a mistake left in settings --> timers --> ZONE CLEANING SCHEDULE (BETA) --> _in pop-up._ 

Screenshot:
![image](https://user-images.githubusercontent.com/5955795/71002321-ac642580-20e7-11ea-95e0-fcadee2cf1ea.png)

**Comment by rand256** [comment]
Yeah, that nice error was already fixed in 4d1ecfc.

**Comment by erkexzcx** [comment]
Hey. That code line, instead of this:
```
 <label for="zoned_timer_thu" class="center" data-i18n="common.days.4">Thursday</label> 
```
should be this:
```
 <label for="zoned_timer_thu" class="center" data-i18n="common.days.3">Thursday</label> 
```

The mistake is seem to be still there. 🤔

**Comment by rand256** [comment]
Can't agree on [that](https://github.com/rand256/valetudo/blob/4d1ecfc5065158b2cb298369bffd274c77c64b6f/client/settings-timers.html#L68). It will be included in 0.8.1 when its ready.
